src/QMCDrivers/TauParams.hpp

```diff
@@ -0,0 +1,65 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2022 QMCPACK developers.
+//
+// File developed by: Ye Luo, yeluo@anl.gov, Argonne National Laboratory
```

Comment:
@ye-luo didn't write this.
Put my name, yours and we have been writing
`// Refactored from MCCoords.hpp`
instead of File created by.

I think that is more meaningful when you're tearing up or porting a well known file.
The name is better.

Reply:
I did the refactoring. So I will take care of this.